### ISSUE TITLE
should we gitignore wrangle.toml? 

### ISSUE THREAD

**Issue by krishFred** [100xdevs-cohort-2/week-13#4]
I have a query should we need to gitignore the wrangle.toml because we are exposing to the git hub and prisma Accelerate is pretty much update which one should we needed to follow
![Screenshot from 2024-08-29 12-23-25](https://github.com/user-attachments/assets/5370d5e2-e7f0-4f39-a740-ef5cad17ce5c)

**Comment by pantharshit007** [comment]
yeah as far as i researched its good to put `wrangle.toml` in gitignore and if you want to check if it will work in edge env run `npm wrangler deploy` and check

**Comment by Raj-G07** [comment]
We can use .dev.vars and store wrangler.toml variable in it , wrangler automatically pick up 
 (-) [vars] 
 (-) DATABASE_URL = ""
 (+) main = src/index.ts
But in prod we can use command like "wrangler secret put VAR_NAME"         And only ignore .dev.vars
